src/main/drivers/pwm_output.h

```diff
@@ -146,6 +151,7 @@ typedef struct {
     volatile bool isInput;
     volatile bool hasTelemetry;
     uint16_t dshotTelemetryValue;
+    uint16_t dshotTelemetryDeadtime;
```

Comment:
Do we know we'll never have an overrun here? Since we are comparing to a `timeDelta_t`, we can probably afford to make this a `timeDelta_t` too.